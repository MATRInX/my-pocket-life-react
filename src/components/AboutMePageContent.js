import React from 'react';

const AboutMePageContent = () => (
  <div>
    AboutMePageContent content...
  </div>
);

export default AboutMePageContent;