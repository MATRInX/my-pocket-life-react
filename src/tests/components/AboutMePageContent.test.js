import React from 'react';
import { shallow } from 'enzyme';
import AboutMePageContent from '../../components/AboutMePageContent';

test('should correctly render AboutMePageContent content', () => {
  const wrapper = shallow(<AboutMePageContent />);
  expect(wrapper).toMatchSnapshot();
});