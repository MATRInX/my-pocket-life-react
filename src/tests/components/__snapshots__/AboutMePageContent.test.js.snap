// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render AboutMePageContent content 1`] = `
<div>
  AboutMePageContent content...
</div>
`;
